src/ast-utils.ts

```diff
@@ -15,32 +15,39 @@
 import * as dom5 from 'dom5';
 import {ASTNode} from 'parse5';
 
-export default class ASTUtils {
-  static prepend(parent, node) {
-    if (parent.childNodes && parent.childNodes.length) {
-      dom5.insertBefore(parent, parent.childNodes[0], node);
-    } else {
-      dom5.append(parent, node);
-    }
+export function prepend(parent, node) {
```

Comment:
noImplicitAny was set to false so I set it to true and also strictNullChecks to true.  And then I type'd all the things.